source/guide/3.2/quickstart.md

```diff
@@ -68,22 +67,12 @@ possible by running the `bcdedit /set hypervisorlaunchtype off` command (reboot
 
 # Step by Step Walkthrough
 
-## Step 1: Download and "Up" your Vagrant Box
+## Step 1: Download Vagrantfile and run VM
 
-The first thing you'll need to do is download the Vagrant box which contains the Cloudify manager and CLI, and the Vagrantfile to run it.
+The first thing you'll need to do is download the Vagrantfile which creates the Cloudify manager and CLI.
```

Comment:
Well.. I think "...containing the Cloudify Manager and CLI" is rather understandable. I'm just being pedantic. It doesn't REALLY matter.